.github/jobs/napijsi.yml

```diff
@@ -0,0 +1,36 @@
+parameters:

```

Comment:
I would name this yml `uwp_napijsi.yml` to be more consistent with other templates.